### ISSUE TITLE
Verify version of open-humans-tornado-oauth2 in production is 2.1.0

### ISSUE THREAD

**Issue by beaugunderson** [biocore/american-gut-web#548]
As part of #539 I upgraded open-humans-tornado-oauth2 to 2.1.0; it looks like I'm getting an error in production using the test account that indicates that it's still at the previous version (if you could forward me the recent tracebacks from the test account generated from /authed/open-humans/ that would also be helpful though!)

For future reference, are packages referenced in `setup.py` automatically updated with each production deploy? (If not I'll just add a note about any required package updates)

cc @madprime

**Comment by beaugunderson** [comment]
Hi @mortonjt; Embriette said you may be able to take a quick look at this, is that possible? :)

**Comment by mortonjt** [comment]
Sure, I can take a look at it tonight.

**Comment by EmbrietteH** [comment]
Thanks Jamie-I appreciate it!

**Comment by beaugunderson** [comment]
@mortonjt did you have a chance to check the version in production? :)

**Comment by madprime** [comment]
Hi @EmbrietteH , @mortonjt Sorry to pester, I wanted to let you know that it would be wonderful to have this done.

Having the American Gut interaction is a big blocker for Open Humans right now, and apparently this is the cause. :( We're otherwise ready to reach out to PGP participants, many of whom could get their American Gut accounts connected.

**Comment by EmbrietteH** [comment]
Thanks for the ping-and sorry for the delay! @mortonjt is having a bit of
trouble working on this as he is still getting familiar with the system and
where everything resides; you can try to ping @squirrelo-he is on vacation
but was responsive today so he might be able to take a quick look!

Embriette R. Hyde, PhD
Postdoctoral Scholar
Project Manager, American Gut Project
Rob Knight Lab
Department of Pediatrics
University of California, San Diego
Phone: 832-845-7532

**Comment by beaugunderson** [comment]
would be a huge help if you could take a look at this @squirrelo :)

**Comment by squirrelo** [comment]
Email is out to our sys admin and should be updated soon. @mortonjt can close this when it is done.

**Comment by madprime** [comment]
Thanks so much folks!! We very much appreciated it, I hope your vacation is good @squirrelo! :)

**Comment by beaugunderson** [comment]
@squirrelo @mortonjt any update on this from the sysadmin?

**Comment by mortonjt** [comment]
Hi @beaugunderson .  Our system admin just installed the newest version.  The version is now definitely 2.1.0.

If you run into any problem, could your post your error messages?

**Comment by squirrelo** [comment]
I’ll update here when I do a re-deploy. We are currently waiting on another pull request, but once it is merged I can restart the servers.

**Comment by squirrelo** [comment]
Re-deploy is in place with upgraded package. Monitoring system now, but looks good.
